src/chigame/api/serializers.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+
+from chigame.games.models import Game
+
+
+class GameSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Game
+        fields = ("name", "description", "min_players", "max_players")
```

Comment:
As of now, neither the list nor detail views include the necesary information to uniquely identify a given game.

You don't include the primary key, nor an API url for games.
This means that once a user has listed games, they have no way to actually access the individual detail view or send actions like DELETE or PATCH.

Add in at least the primary key as a field such that the necessary identificatory information is always associated with each game.

You can think of this as, API users don't have access to a nice watchbuddies style game listing where you can click on links to do things like edit and delete. 
Thus you need to give the endpoint user all the info they need to reconstruct that kind of usually internally handled work that GUI and backend can do.

Reply:
I added the `id` field to the GameSerializer so that the user will also receive the primary key of games, when getting the list of games at the endpoint (/api/games). This will allow the user to use the `id` to then get the detail view of a specific game.